packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/step.st

```diff
@@ -0,0 +1,8 @@
+simulation
+step
+	"Step only if the simulation is running."
+	
+	self isSimulationRunning
+		ifTrue: [
+			super step.
```

Comment:
Just as a note for the xcom, the method you call here is empty. Should we still call it?